File: code-coaches/easy/fruit-bowl/fruit_bowl.cpp
#include <iostream>

int get_apple_pies(int fruit_count)
{
    int apple_count = fruit_count / 2;
    return apple_count / 3;
}

int main()
{
	int fruit_count;
	std::cin >> fruit_count;

	std::cout << get_apple_pies(fruit_count);

	return 0;
}

File: code-coaches/easy/fruit-bowl/fruit_bowl.rb
def get_apple_pies(fruit_count)
    apple_count = fruit_count / 2
    return apple_count / 3
end


def main
    fruit_count = gets.chomp.to_i
    puts get_apple_pies(fruit_count)
end


if __FILE__ == $0
  main
end
